A few small fixes

jszwedko opened the pull request:
Including:

* Initialize new default event structure; avoids trying to access non-existant properties before the event is loaded
* Start map zoomed in on San Francisco before locating, rather than showing the whole world
* Unset locate query parameter when user is located; avoiding the situation where a user clicks an event, comes back to the map, and is then relocated.

jszwedko committed:
Unset locate query parameter when user is located

Avoids the situation where a user clicks an event, comes back to the
map, and is then relocated.

jszwedko committed:
Start map zoomed in on San Francisco before locating

Rather than showing the whole world

jszwedko committed:
Initialize new default event structure

Avoids trying to access non-existant properties before the event is
loaded

jszwedko committed:
standardjs linting

akegan reviewed (approved):
looks great, maybe just update the default zoom to zoom out a bit more so you can see the San Francisco Label or something. I think 13 or 14 looks good

akegan commented:
Ok one weird bug is that when you drop the locate, it seems to update the query params and the map moves slightly. check out this gif. 
![weird_map_movement5](https://user-images.githubusercontent.com/11825994/45190213-a481ba00-b1f1-11e8-8ef7-c91734f05c98.gif)

jszwedko committed:
Fix bounds calculation

Should have been using 90 - angle for northEastBound, but I switched to
calculating the other angle to make the arithmetic clearer (can still
use angle for bearing of northEastBound and then -180 + angle for
southWestBound)

jszwedko commented:
@akegan that bit is intentional albeit admittedly confusing. The way the "Find by me" works from the start page is to set a query parameter so that, when the map loads, it locates the user. In your case, the map default location was close to yours.

This is to work around the lack of ability to set the initial location for the locate control (which draws the user location). I have some more details on https://github.com/domoritz/leaflet-locatecontrol/issues/209

I definitely welcome ideas of ways to make this less confusing though.

However! Your gif made me realize the bounds calculation was incorrect, again, as I was calculating the bearing for the north east corner incorrectly. Fixed in https://github.com/SFDigitalServices/neighborhood-noticing/pull/19/commits/1d62a57a48ddb00d3eaeffa5bb0deeaceb706390

jszwedko commented:
Gonna merge this just to get it in ahead of the share-out, but happy to incorporate additional feedback!
